assets/src/application/ui/display/ItemCount/style.scss

```diff
@@ -6,6 +6,14 @@
 .ee-item-count {
 	&__wrapper {
 		position: relative;
+		min-width: 55px;
+		.ee-editor-details-reg-url-link {
+			align-items: center;
+			display: flex;
+			height: calc(var(--ee-icon-button-size) + 2px);
+			justify-content: center;
+			width: calc(var(--ee-icon-button-size) + 2px);
+		}
```

Comment:
hmmmm... 
this should be collocated with that date reg url link component instead of being added here. 
ie: this is domain specific logic leaking into the app layer (it happens)

IF the issue this fix was added for could potentially be more common and apply to other components, then maybe create a new class for applying these styles and then use a new prop to control that

Reply:
updated: https://github.com/eventespresso/event-espresso-core/pull/2727/commits/86fcae34ef97a29fef252468123c930d9c2a9570